.github/workflows/ci.yml

```diff
@@ -16,8 +16,10 @@ jobs:
       run: pip install -r tools/requirements.txt
     - name: check .rst document syntax
       run: tools/check-rst-syntax.sh
-    - name: generate and check intrinsic specs (rst)
+    - name: generate and check intrinsic specs (md)
       run: tools/generate-intrinsics-specs.sh
+    - name: install pandoc and dependencies
+      run: sudo apt-get -y install pandoc texlive-latex-base texlive-fonts-recommended texlive-fonts-extra texlive-latex-extra inkscape
```

Comment:
I think the `--no-install-recommends` is still missing?

Reply:
I noticed yes